### ISSUE TITLE
Conch dies when there's no ~/.ssh directory

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7489]
|<img alt="dp's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| dp reported|
|-|-|
|Trac ID|trac#187|
|Type|defect|
|Created|2003-08-08 16:55:32Z|

<details><summary>Searchable metadata</summary>

```
trac-id__187 187
type__defect defect
reporter__dp dp
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1060361732000000 1060361732000000
changetime__1060374668000000 1060374668000000
version__ 
owner__ 
cc__dp cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="dp's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|dp commented|
|-|-|

```
#!html
<pre>
If there is no ~/.ssh directory, connecting to a host will attempt to 
append to ~/.ssh/known_hosts and fail.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="z3p's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|z3p commented|
|-|-|

```
#!html
<pre>
fixed in conch:1.58

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
